Makefile

```diff
@@ -155,11 +155,16 @@ $(foreach lib,$(VALID_LIBRARIES),$(call include_lib,$(lib)))
 $(foreach proj,$(VALID_PROJECTS),$(call include_proj,$(proj)))
 
 IGNORE_CLEANUP_LIBS := CMSIS FreeRTOS STM32F0xx_StdPeriph_Driver unity FatFs
+IGNORE_PY_FILES := lint.py
+# Find all python files excluding library files in project env (./venv)
+FIND_PY_FILES:= $(shell printf "! -name %s " $(IGNORE_PY_FILES) | xargs find -path ./venv -prune -o -name '*.py')
+AUTOPEP8_CONFIG:= -a --max-line-length 100 -r
```

Comment:
Could you also add the whole `unity` directory as ignored? It contains a bunch of python files which aren't *really* owned by us (they're part of the unity test library) and formatting them would probably cause merge conflicts.

(Also, please put a space before the `:=`)